src/main/frontend/components/settings.cljs

```diff
@@ -127,10 +127,10 @@
    {:style {:box-shadow "0 4px 20px 4px rgba(0, 20, 60, .1), 0 4px 80px -8px rgba(0, 20, 60, .2)"}}
    [:div {:style {:margin "12px" :max-width "500px"}}
     [:p.text-sm
-     "The left side shows outdenting with the default setting, and the right shows outdenting with logical outdenting enabled. "
```

Comment:
@sprocketc Interesting case where whitespace visually helps the tooltip. Left it as is in the translation for now